db.go

```diff
@@ -60,11 +59,9 @@ type Options struct {
 
 	// The timestamp range of head blocks after which they get persisted.
 	// It's the minimum duration of any persisted block.
+	// It should divide 2h cleanly and should be less than or equal to 2h. Default 2h.
 	MinBlockDuration uint64
```

Comment:
As per above suggestion, we probably want to pass in the full ranges as config options instead of this value. The first range is than equivalent to `MinBlockDuration`. As said there, we also shouldn't make assumptions about stuff like 2h.

At least for bench/debugging purposes it probably would be a good idea to still allow passing in short initial ranges via Prometheus. Maybe a hidden `--debug.storage.tsdb...` flag?